Dockerfile.upload_failures

```diff
@@ -12,4 +12,11 @@ COPY backend/corpora/common ./chalicelib/backend/corpora/common
 COPY backend/corpora/dataset_processing ./chalicelib/backend/corpora/dataset_processing
 RUN mkdir -p ./.chalice
 
+ARG HAPPY_BRANCH="unknown"
```

Comment:
We also have `make` targets for building images and working with docker-compose. Are we going to (or going to want to?) add these labels to containers in local dev?

Reply:
Current thought was not to do it for local dev. In our specific case, we run with a dev server locally and with a single replica so this build ID wouldn't come into use there. That said, I am going to modify the Dockerfiles to set HAPPY_COMMIT's default to empty string instead of "unknown" to allow next to resort to its normal random ID.